src/_data/side-nav.yml

```
@@ -212,7 +201,13 @@
       - title: Dart DevTools
         permalink: /tools/dart-devtools
       - title: DartPad
-        permalink: /tools/dartpad
+        children:
+          - title: Overview
+            permalink: /tools/dartpad
+          - title: DartPad in tutorials
+            permalink: /resources/dartpad-best-practices
```

If you're moving this, let's avoid the redirect:

```suggestion
            permalink: /tools/dartpad/dartpad-best-practices
```